cartography/intel/aws/secretsmanager.py

```diff
@@ -0,0 +1,78 @@
+import logging
+from typing import Any
+from typing import Dict
+from typing import List
+
+import boto3
+import neo4j
+
+from cartography.util import dict_date_to_epoch
+from cartography.util import run_cleanup_job
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+
+@timeit
+def get_secret_list(boto3_session: boto3.session.Session) -> List[Dict]:
+    client = boto3_session.client('secretsmanager')
+    paginator = client.get_paginator('list_secrets')
+    secrets: List[Any] = []
+    for page in paginator.paginate():
+        secrets.extend(page['SecretList'])
+    return secrets
+
+
+@timeit
+def load_secrets(neo4j_session: neo4j.Session, data: List, current_aws_account_id: str, aws_update_tag: int) -> None:
```

Comment:
Also here (nitpicking again, sorry), let's make this as `data: List[Dict]` rather than just `data: List`.